sdk/src/logs/multi_log_processor.cc

```diff
@@ -129,10 +129,8 @@ bool MultiLogProcessor::Shutdown(std::chrono::microseconds timeout) noexcept
   }
   for (auto &processor : processors_)
   {
-    if (!processor->Shutdown(std::chrono::duration_cast<std::chrono::microseconds>(timeout_ns)))
-    {
-      result = false;
-    }
+    result |=
```

Comment:
Probably `||=` instead of `|=` operator?

Reply:
Never mind, seems `||=` is not a valid operator.

Reply:
This is the same as `MultiSpanProcessor`. Maybe we should only check the timeout and always return true when it do not reach timeout.
I have added `DurationUtil` to fix the overflow problem when using and converting duration types in #1209 . We can modify `Shutdown` and `ForceFlush` later.